clang/test/lab1/veselov_ilya/test.cpp

```diff
@@ -0,0 +1,37 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/PrintClassDescPlugin%pluginext -plugin print-class %s 1>&1 | FileCheck %s
+
+// CHECK: Empty
+class Empty {};
+
+// CHECK: MyStruct
+struct MyStruct{
+    // CHECK-NEXT: |_a
+    int a;
+    // CHECK-NEXT: |_b
+    int b;
+};
+
+// CHECK: MyClass
+class MyClass {
+    // CHECK-NEXT: |_a_c
+    int a_c;
+    // CHECK-NEXT: |_b_c
+    float b_c;
+    // CHECK-NEXT: |_c_c
+    double c_c;
+};
+
+// CHECK: outerClass
+class outerClass {
+    // CHECK: innerClass
+    class innerClass {
+        //CHECK-NEXT: |_var
+        float var;
+    };
+};
+
+// CHECK: TemplateClass
+template<typename T> class TemplateClass {

```

Comment:
Could you please also add test with instantiation of this class for a concrete type?